### PR TITLE
ydcv: update to 0.6.2

### PR DESCRIPTION
**Pull request by ideal**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
==> /usr/local/Cellar/ydcv/0.6.2/bin/ydcv 你好
Traceback (most recent call last):
  File "/usr/local/Cellar/ydcv/0.6.2/bin/ydcv", line 96, in <module>
    _re_non_english = re.compile(r'[^\w]', re.ASCII)
AttributeError: 'module' object has no attribute 'ASCII'
Error: ydcv: failed
An exception occurred within a child process:
  Test::Unit::AssertionFailedError: <0> expected but was
<1>.
```

**Comment by ideal** [comment]
That is because ydcv references an attribute of `re` but it is only available in python 3.
I'll check how to fix it.

File: Formula/ydcv.rb [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+  include Language::Python::Virtualenv
+
   desc "YouDao Console Version"
   homepage "https://github.com/felixonmars/ydcv"
-  url "https://github.com/felixonmars/ydcv/archive/0.6.1.tar.gz"
-  sha256 "0a01fed567a1045cf6c81da9532ed07852f99dc93c862f59255fda74aab05714"
+  url "https://github.com/felixonmars/ydcv/archive/0.6.2.tar.gz"
+  sha256 "45a237fba401771c5ad8455938e6cf360beab24655a4961db368eb2fbbbfb546"
 
   bottle :unneeded
```
Comment (fxcoudert) [inline Formula/ydcv.rb]: Remove the `bottle` line

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/ydcv.rb (hunk 1 above)
Comment (ideal) [reply Formula/ydcv.rb]: OK, removed.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @ideal

**Pull request closed without merging**